sdk/lib/ucrt/conio/getch.cpp

```diff
@@ -229,7 +229,7 @@ static int get_next_getch_pushback_buffer()
     return ret_val;
 }
 
-extern "C" extern intptr_t __dcrt_lowio_console_input_handle;
+extern "C" intptr_t __dcrt_lowio_console_input_handle;
```

Comment:
In `sdk/lib/ucrt/conio/cgets.cpp` you used instead to add extra curly brackets to enclose the innermost `extern type variable;` part. Isn't it possible to do that here as well?
`+` Same question for the other files below where such a change has been done that way.

Reply:
The ones in cgets.cpp are declarations and requires the curly braces and an extra `extern`. This one is a definition, so it doesn't need extra braces. I can add them for consistency though.